Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserModel.h

```diff
@@ -84,10 +85,14 @@ namespace AzToolsFramework
             AZStd::shared_ptr<RootAssetBrowserEntry> GetRootEntry() const;
             void SetRootEntry(AZStd::shared_ptr<RootAssetBrowserEntry> rootEntry);
 
+            AssetBrowserFilterModel* GetFilterModel() const;
+            void setFilterModel(AssetBrowserFilterModel* filterModel);
```

Comment:
update: Sorry if this is overriding a Qt function it can be lowercase but then we should add `override` here, if it's not, then update to `UpperCase`